nio/client/async_client.py

```diff
@@ -1233,6 +1235,34 @@ async def devices(self) -> Union[DevicesResponse, DevicesError]:
 
         return await self._send(DevicesResponse, method, path)
 
+    @logged_in
+    async def update_device(
+            self,
+            device_id: str,
+            content: Dict[str, str]
+    ) -> Union[UpdateDeviceResponse, UpdateDeviceError]:
+        """Update the metadata of the given device.
```

Comment:
It'd be handy if you gave more detail about what "meta data" is; you could either write it all our (if it's just device_name) or link to elsewhere in the docs.

Reply:
I looked into the [specs][0] and *currently* `display_name` seems to be the only "metadata".
But if i write it out this needs to be updated every time the specs change.
Maybe we should link to the matrix [specs][0]?

[0]: https://matrix.org/docs/spec/client_server/latest#put-matrix-client-r0-devices-deviceid